config/locales/en.yml

```diff
@@ -335,15 +335,16 @@ en:
         title: Reset password
       mfa:
         multifactor_auth: Multifactor authentication
-        disabled: You have not yet enabled multifactor authentication.
+        disabled: You have not yet enabled multifactor authentication. Please refer to <a href="https://guides.rubygems.org/setting-up-multifactor-authentication">RubyGems MFA guide</a> for more information on these levels.
```

Comment:
What about to add target blank to open that in new window/tab since in theory it is on settings edit page and some work can be lost?

Reply:
I agree with this if we find other uses of target blank in the codebase but otherwise unless there’s actually potential loss of state I wouldn’t bother.